### DISCUSSION TITLE
Which 40G network cards are supported by oracle linux 9.3?

### DISCUSSION THREAD

**Discussion by bioteseo** [Security-Onion-Solutions/securityonion#12373] [2.4] [answered]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

64

### Storage for /

200G

### Storage for /nsm

10T

### Network Traffic Collection

tap

### Network Traffic Speeds

more than 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi,
I need to install 40G network cards on my sensors on which I installed SO 2.4.40 using the SO cd. So the distro is oracle linux 9.3.

Which 40G network cards are supported by oracle linux 9.3? Is there any documentation on this?

Is anyone already using such cards?

Thanks in advance

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by robbiemarshall** [comment, marked as answer]
Here is a hardware component compatibility guide for Oracle Linux:  https://linux.oracle.com/Component_Compatibility_Guide.pdf
